settings.yaml

```diff
@@ -0,0 +1,25 @@
+HistoryService:
+    Db: 
+        PostgresDataConnString:
+            settings-key: HistoryService-PostgresDataConnString
```

Comment:
add types?

Reply:
afaik, we do not have types for PostgresSQL